### ISSUE TITLE
Auditor Role

### ISSUE THREAD

**Issue by jneuner-mitre** [mitre/heimdall2#485]
As a Security Control Assessor, I need to be able to review evidence of control compliance while adhering to the principle of least privilege.

Read-only user access control for circles seems like the most common use case here.

**Comment by Bialogs** [comment]
@jneuner-mitre this request seems to be asking for read only access to Evaluations that another user grants you?

**Comment by jneuner-mitre** [comment]
Yep.  I want to grant an SCA read-only access to a circle/group of evaluations for a specific system component target.  Then in my [IA management tool](https://en.wikipedia.org/wiki/EMASS), I can reference a system component and provide a link to the appropriate circle.

**Comment by Amndeep7** [comment]
@jneuner-mitre @aaronlippold is this functionality still being requested?
